keyboards/ep/96/readme.md

```diff
@@ -0,0 +1,15 @@
+# ep96
+
+![ep96](https://imgur.com/PZ2C7a8)
```

Comment:
This should link directly to an image, rather than just an Imgur page (if you're only linking one image).

```suggestion
![ep96](https://i.imgur.com/PZ2C7a8.jpg)
```